### PR TITLE
Making error message human readable for shopify customers

### PR DESCRIPTION
**Pull request by MauricioMurga**
Making error message human readable for shopify customers.

### CODE REVIEW

**Comment by j-mutter** [comment]
@MauricioMurga is there documentation somewhere I can see to confirm this is the correct field to use? I don't see it anywhere in [these docs](https://www.conekta.io/en/docs/api).

File: lib/active_merchant/billing/gateways/conekta.rb [hunk 1]
```diff
@@ -197,8 +197,9 @@ def response_error(raw_response)
       end
 
       def json_error(raw_response)
-        msg = 'Invalid response received from the Conekta API.'
-        msg += "  (The raw response returned by the API was #{raw_response.inspect})"
+        #msg = 'Invalid response received from the Conekta API.'
+        #msg += "  (The raw response returned by the API was #{raw_response.inspect})"
```
Comment (j-mutter) [inline lib/active_merchant/billing/gateways/conekta.rb]: These can be deleted. Please don't submit PRs with commented out code, it just means we have to go in and remove it when we merge.

**Comment by MauricioMurga** [comment]
@j-mutter, message_to_purchaser documentation can be found here: https://www.conekta.io/en/docs/api#errors

**Comment by j-mutter** [comment]
Oh weird, I must have typo'd my search when I first looked. Thanks :)

**Comment by MauricioMurga** [comment]
@j-mutter, no problem :)

**Comment by ntalbott** [comment]
Looks like this since got merged elsewhere.

**Pull request closed without merging**